hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/impl/ZookeeperFederationStateStore.java

```diff
@@ -123,16 +126,24 @@
 import static org.apache.hadoop.util.curator.ZKCuratorManager.getNodePath;
 
 /**
- * ZooKeeper implementation of {@link FederationStateStore}.
+ * ZooKeeper's implementation of {@link FederationStateStore}.
```

Comment:
I would leave it as it was.

Reply:
Thank you very much for your help in reviewing the code! I will fix it.